lib/portlayer/exec/container.go

```diff
@@ -377,15 +382,23 @@ func (c *Container) Signal(ctx context.Context, num int64) error {
 	return c.startGuestProgram(ctx, "kill", fmt.Sprintf("%d", num))
 }
 
-func (c *Container) LogReader(ctx context.Context) (io.ReadCloser, error) {
+func (c *Container) stopped() {
```

Comment:
Just wanted a generic method that'd be called when an container is stopped, to
avoid sandwiching duplicate code into the same two places in the future.  I can rename it.